eng/pipelines/steps/release-build-steps.yml

```diff
@@ -289,7 +291,9 @@ steps:
             -github-app-client-id '$(BotAccount-bot-for-go-client-id)' \
             -github-app-installation '$(BotAccount-bot-for-go-installation)' \
             -github-app-private-key '$(BotAccount-bot-for-go-private-key)' \
-            -github-reviewer-pat '$(GitHubPATReviewer)' \
+            -github-reviewer-app-client-id '$(BotAccount-review-bot-for-go-client-id)' \
+            -github-reviewer-app-installation '$(BotAccount-review-bot-for-go-installation)' \
+            -github-reviewer-app-private-key '$(BotAccount-review-bot-for-go-private-key)' \
```

Comment:
Have you run a Go images update using this flow? Otherwise should revert this chunk.

Reply:
I ran one in my test fork which worked? Is that enough?

Reply:
Given all the issues we've had are due to permissions, not code, I don't think so.

Reply:
okay removed these changes (for now)